src/EditorFeatures/CSharpTest/Debugging/LocationInfoGetterTests.cs

```diff
@@ -83,7 +83,7 @@ void Method()
     {
     }$$
 }
-", "Class.Method()", 2);
+", "Namespace.Class.Method()", 2);
```

Comment:
Why this change?

Reply:
This is correct.  The namepace name shoudl be appended. but it wasn't because the walker wasn't seeing the file-scoped-namespace and was only checking for normal namespaces.